libs/providers/src/lib/language-swithcer/data.ts

```diff
@@ -0,0 +1,129 @@
+import { LangItems } from './types';
```

Comment:
please don't put this file in providers, we don't need it. each application should provide their own languages. we shouldn't provide any suggestions. thanks 🙏 
please pass the hk langs in `deriv-hk` app.